src/components/modules/modificationsObserver.ts

```diff
@@ -18,7 +18,7 @@ export default class ModificationsObserver extends Module {
    *
    * @type {number}
    */
-  public static readonly DebounceTimer = 450;
+  public DebounceTimer = this.config.onChangeDebounceTime || 450;
```

Comment:
The same as above

Reply:
@gohabereg please refer to commit b8137a1c312e837eb44eb08a35415dc99e841a81